vendor.conf

```diff
@@ -50,5 +50,6 @@ github.com/vishvananda/netns 604eaf189ee867d8c147fafc28def2394e878d25
 golang.org/x/crypto 558b6879de74bc843225cde5686419267ff707ca
 golang.org/x/net 7dcfb8076726a3fdd9353b6b8a1f1b6be6811bd6
 golang.org/x/sys 07c182904dbd53199946ba614a412c61d3c548f5
+github.com/golang/sync fd80eb99c8f653c847d294a001bdf2a3a6f768f5
```

Comment:
This uses the wrong (non-canonical) import path; see https://github.com/docker/libnetwork/pull/2111 (and https://github.com/moby/moby/pull/36589/files#r174434213)